### ISSUE TITLE
Reorganizing the landing page

### ISSUE THREAD

**Issue by bedroesb** [elixir-europe/rdmkit#754]
With our UX/UI expert looking at the website it was more than ever clear we have to get rid of the block of text on the landing page + people need to be redirected to the tiles with an obvious button

[UX input](https://docs.google.com/document/d/1voKjx6trkDLwvFDlBnH9u07EJyzpXyqbVbaM5cygrmY/edit#heading=h.fg0ux0hduedh)

**Comment by bedroesb** [comment]
Proposal by @floradanna :


RDMkit - The Research Data Management toolkit for Life Sciences
- Data management best practices and guidelines applicable to research projects from the beginning to the end.
- Information, pointers and examples to make your data FAIR — Findable, Accessible, Interoperable and Reusable – by design.
- Tools, resources and training materials used by life sciences research communities for your RDM.

**Comment by martin-nc** [comment]
I'd even go shorter and put Flora's text as the first part of the About page. Maybe something like: "The Research Data Management kit (RDMkit) for LIfe Sciences helps you follow data management best practices and make your data FAIR: Findable, Accessible, Interoperable and Reusable." Then maybe a link to the About page to find out more. 

Here is a comparison of (L to R) how the home page is now, then with Flora's text, then with the shorter text above. Notice how the content below is more visible in the last one, and people can interact with it quicker (e.g. click on the life cycle wheel).

<img width="1000" alt="rdm-screenshots" src="https://user-images.githubusercontent.com/15261745/146519168-c9ee9cd7-c675-4d76-ab7e-1ccc06e81fd5.png">

**Comment by floradanna** [comment]
- I looked at the suggestions given by the UX expert at BH2021 and I made a proposal (see below).
  - the entire tile (data life cycle, your tasks etc) could be clickable and could correspond to the first page of each section (for instance, Plan for data life cycle, Compliance and monitoring for Your tasks etc), at least until we make a page for each section.
- I also realised that we probably don't need a landing page, but we can just have a nice home page that goes directly to the point.
- I also noticed some inconsistencies that could be easily harmonised (I think):
  - Home button in header --> Home button in side bar _is missing_ --> _Page title is missing_
  - About button in header --> About button in side bar (_ok_) --> Page title could be "About RDMkit" (since the titles seem to be different from the side bar names elsewhere)
  - Contribute button in header --> Contribute button in side bar _is called differently! It should be "Contribute" for consistency_ --> Page title _ok_
  - Contact button in header --> no need to show the side bar of the home page in this page (it doesn't make much sense)
<img width="816" alt="Screenshot 2022-02-01 at 19 38 24" src="https://user-images.githubusercontent.com/51486716/152122465-0d7af04c-36f8-4480-b0f9-f6028a766858.png">
<img width="815" alt="Screenshot 2022-02-01 at 19 38 37" src="https://user-images.githubusercontent.com/51486716/152122476-d9126c8a-bbc6-45ef-86ed-d1d97835d13a.png">

**Comment by martin-nc** [comment]
> the entire tile (data life cycle, your tasks etc) could be clickable and could correspond to the first page of each section

So if you clicked on "Your role" you would go to "Researcher", and if clicked on "Your Tasks" you would go to "Compliance monitoring & measurement" etc?

> I also realised that we probably don't need a landing page, but we can just have a nice home page that goes directly to the point.

So things would work like above (if you clicked on "Your role" you would go to "Researcher" etc)?

> I also noticed some inconsistencies...

Well, here's my take FWIW:

I think the problem is that there is a flaw in the site theme: it's not clear which is the main menu of the site, and what the main sections are. Traditionally you'd have a main menu that is always there. It shows all the main sections of the site, and orientates you in the site. The main menu would collapse into a hamburger menu on a mobile device. 

In this theme, it's the top menu that collapses into a hamburger menu and stays the same across the site. This is the main menu, not the side menu. So really the top menu should have a label for each section of the site that has its own side menu, plus a link for the home page. For the RDMkit: Home, Manage your data, Contribute, About. (We haven't got a label for the main section of the site, with the Data Life Cycle, Your role etc in. I've just called it "Manage your data" so you can get the idea.) 

Then when you move across the top menu the side menu changes to match the section you're in. Ideally, I would put "Contact" and "GitHub" in a secondary menu at the top of the header. You wouldn't have the side bar on the home page because the home page is a page, not a section. The home page doesn't have subpages, so it doesn't need a submenu.

So putting the 'Home' button in the side bar could be confusing. It might make the side bar look like the main menu, which it isn't. For me "About" and "Contribute" work more as expected. You click on the link at the top and get a section menu at the side (or before the content on mobile). The submenu shows the pages of that section. 

The alternative approach is to make the sidebar the main menu and include all links ("Contribute", About" etc). This would  make the side menu the same across the whole site.

So anyway:

> Home button in header --> Home button in side bar is missing 

I don't think it should be there, but I don't the sidebar should be on the home page at all! 

> About button in header --> About button in side bar (ok) --> Page title could be "About RDMkit"

I'd keep the labels consistent.

> Contribute button in header --> Contribute button in side bar is called differently! It should be "Contribute" for consistency

Yes.

> Contact button in header --> no need to show the side bar of the home page in this page

Yes, "Contact" is like the home page. It's a page, not a section with subpages.

**Comment by floradanna** [comment]
> So if you clicked on "Your role" you would go to "Researcher", and if clicked on "Your Tasks" you would go to "Compliance monitoring & measurement" etc?  
> So things would work like above (if you clicked on "Your role" you would go to "Researcher" etc)?

Yes, just until we take the time to create a page for each section. It could be a way to make the tiles clickable and go from the tiles in "landing page" to the section in the side bar and unfold the collapsable list under each section. I know it is not ideal, but maybe it is less strange that what we have now, that is 2 ways to unfold the collapsable list under each section: in the side bar and in the tiles (where applicable).

> In this theme, it's the top menu that collapses into a hamburger menu and stays the same across the site. This is the main menu, not the side menu. 

Yes, I agree.

> So really the top menu should have a label for each section of the site that has its own side menu, plus a link for the home page. For the RDMkit: Home, Manage your data, Contribute, About. (We haven't got a label for the main section of the site, with the Data Life Cycle, Your role etc in. I've just called it "Manage your data" so you can get the idea.)

In this case we should create a new "landing page" or what you call "Home" page in your example, maybe also reachable from RDMkit icon, that doesn't have side menu and doesn't show any wheel. I would call this a landing page, but we don't have it.
That is why my proposal is to have directly our current "Home" page = "Manage your data" page. So, no landing page, but the "Home/Manage your data page" directly.

> Then when you move across the top menu the side menu changes to match the section you're in. Ideally, I would put "Contact" and "GitHub" in a secondary menu at the top of the header. You wouldn't have the side bar on the home page because the home page is a page, not a section. The home page doesn't have subpages, so it doesn't need a submenu.

Yes, I agree. I don't know how "Contact" could be handle technically if we don't make a new landing page.


> So putting the 'Home' button in the side bar could be confusing. It might make the side bar look like the main menu, which it isn't. For me "About" and "Contribute" work more as expected. You click on the link at the top and get a section menu at the side (or before the content on mobile). The submenu shows the pages of that section.

In my proposal, the "Home" button could be renamed with whatever we decide to call the "Manage your data" section. So, we could rename our current "Home" to "Manage your data" and have this button on the side bar as well. I agree that "About" and "Contribute" work more as expected. "Manage your data" button will then work as About and Contribute.


> So anyway:
> > Home button in header --> Home button in side bar is missing
> I don't think it should be there, but I don't the sidebar should be on the home page at all!

"Home" button could be renamed as "Manage your data" and be placed on the side bar.
 

Basically I agree with you Martin, I was only proposing a very pragmatic solution: since we don't have a landing page, we could stop pretending we have one and work only with 3 sections, that are "Manage your data" - "About" - "Contribute", by using the "Manage your data" as default/home for RDMkit. I hope I clarified a bit my proposal.

**Comment by Nazeeefa** [comment]
Hi all! Nice discussions. Here to share suggestions for 'Contact' page.

But first, I see that Contact page is not in 'working/test' website https://bedroesb.github.io/rdmkit/how_to_contribute.html is the plan to completely remove it? Since only email address is mentioned in 'Contact' page (on live website), you can replace it with a button in top navigation menu and add (mail/letter) icon - similar to how 'Github' is displayed with text and icon (in top menu). Usually icons are enough to display in top/footer menu, but not so accessible without name/title. The link for 'Contact' button would then be mailto:rdm-editors@elixir-europe.org

It would be nice to have these Contact and Github button links, together with icons, in website footer too.

**Comment by Nazeeefa** [comment]
Yeah I am not a big fan of random mail client being opened upon click, probably shouldn't suggest what I don't prefer :D but good to know many don't prefer it.

Even if it's moved to 'About', it is still important to display quick links/text in top and footer menu. Maybe we can replace 'Contact' with open text (but no link) so e.g. in top/footer menu, it's mail icon and email address written next to it.

**Comment by Nazeeefa** [comment]
Hi again! I had a think about the landing page again, and I think we should go with what @floradanna suggests - here's what to do next:

- Add description next to data life cycle, as shown in the figure kindly shared by @floradanna. We need to think here about a balance between having enough whitespace and not making content + image cramped.
- Make data life cycle steps clickable.
- Instead of 'Site overview' button (as mentioned in #857), it makes sense to have 'Read More' button as shown in the figure by Flora
- Add icons for pages e.g. 'Your Task', 'Tool Assembly' etc (as shown in the figure) while keeping the font type and size for section/page titles same as it is now. 
- In addition, I suggest to make page titles clickable - then you can remove links (as the one underlined in screenshot below) that don't seem necessary
<img width="1355" alt="Screenshot 2022-03-04 at 23 04 55" src="https://user-images.githubusercontent.com/6730853/156848335-24b2b1b7-7456-4007-93ac-6e5f6e538b66.png">

**Comment by Nazeeefa** [comment]
> Yes, just until we take the time to create a page for each section. It could be a way to make the tiles clickable and go from the tiles in "landing page" to the section in the side bar and unfold the collapsable list under each section. I know it is not ideal, but maybe it is less strange that what we have now, that is 2 ways to unfold the collapsable list under each section: in the side bar and in the tiles (where applicable).

Hi @floradanna and all - while I appreciate the idea about making entire tile clickable, I am afraid it's not user friendly (as you also know it's not ideal). This is because tiles are often treated as non-clickable element as compared to, for example, buttons, headings, and drop-down menus. In other words, tile/box as a clickable is not a familiar and common design to most users. Also, in our case, some tiles have 'Show Pages' drop-down and adding multiple clickable elements to a tile can make navigation tricky and less user-friendly. The last point in my previous comment is what I would suggest could be a good alternative here. Happy to know your thoughts and suggestions :) thank you!

**Comment by floradanna** [comment]
Thanks for your input! For me, it is quite difficult to comment only on one aspect of the GUI because I believe that many things depend on other, that are also changing from time to time (for instance, landing page per sub-section, subpages, news on header etc). If something change, it will likely affect other aspects of the GUI.

If we introduce landing page for each sub-section, the description that is now in boxes could go the related landing page. Same for the clickable wheel. Now that we have 4 sections (DM, About, Contribute, What's new), do we want similar "structure" in each section (do we want "read more" in each section?)? There are numerous considerations, so I can only say it depends.

**Comment by bedroesb** [comment]
The description in the boxes now are off course to let people decide whether clicking on the box is relevant for them. If you take that away they blindly have to click on everything. It is in general my major concern in moving to the landing pages: not improving the experience of the website. But I am open for mayor changes concerning how people navigate trough the RDMkit website.

**Comment by martin-nc** [comment]
I came across this Jekyll theme recently: https://jekyll-theme-desk.netlify.app/. What about making the RDMkit home page like this? We bring the search down to the middle of the page, lose the sidebar on the home page, and have tiles to the different sections underneath the search. 

I think one of the problems of the current site is that the user needs to translate the words they use to describe their problem into the terms the website uses. They may not have sufficient knowledge to do that. Even if they did, it may take several clicks to get where they need to go.

In the recent feedback, someone had been asked by a journal to deposit their data somewhere, but the user didn't know where to begin. They clicked on the 'Sharing' stage of the life cycle then got lost. The terms they were looking for were "repository", "research data archive" and "deposition database". They didn't see these anywhere obvious. If they'd put the terms in the search they would have been presented a choice of options, and found what they wanted much quicker. The search allows people to guess the terms they need. The navigation demands that people understand exact terms and how the site uses them. The navigation also takes more clicks!

**Comment by floradanna** [comment]
@martin-nc I like a lot your suggestion (I can already see it!). It would also fit well we the decision to have "landing pages" for each sidebar sub-section (role, task etc) that we made during the last meeting. It would also make the content more easy to navigate/discover, as asked by a user. We also have to improve the visibility of "related pages" in stage page (as suggested by a user and other editors). 


<img width="646" alt="Screenshot 2022-03-31 at 19 27 56" src="https://user-images.githubusercontent.com/51486716/161114908-07bbbb85-9754-4ffc-a46c-dd1ff67e8454.png">

**Comment by bedroesb** [comment]
> I would suggest either have the same wheel on the life cycle page or none.

which same wheel? :) that they are all colored or?
